### DISCUSSION TITLE
Flathub gsoc

### DISCUSSION THREAD

**Discussion by razzeee** [renovatebot/renovate#27087] [Request Help]
### How are you running Renovate?

None

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

Hey there,

this isn't really a question and more of a heads up - we're exploring doing a gsoc this year with a renovate topic connected to flathub integration.

It would be nice to have some contacts on your side, to potentially help out in the progress. We're currently preparing our proposal here https://github.com/flathub-infra/gsoc/pull/4/files

**Comment by rarkins** [comment]
Hi, we're pretty responsive to questions here in discussions. I'm also easy to reach via email. Alternatively if you progress to writing any code then you're also welcome to participate in our contributor slack channels
